src/drivers/nuttx/lv_nuttx_fbdev.c

```diff
@@ -39,6 +39,7 @@ typedef struct {
 
     void * mem;
```

Comment:
need to keep?

Reply:
Yes, to store the physical fb memory address.